static/js/popovers.js

```diff
@@ -456,6 +457,7 @@ export function toggle_actions_popover(element, id) {
     const $elt = $(element);
     if ($elt.data("popover") === undefined) {
         const message = message_lists.current.get(id);
+        const message_deleted = message.content === "<p>(deleted)</p>";
```

Comment:
Extracting this variable is a good refactor; could be a fine prep commit to merge early, but it's sufficiently localized that it's not super important.

Reply:
NOCHG - deleted and no longer relevant to this PR, since we *are* showing reportmsg option on deleted messages, as per https://github.com/zulip/zulip/pull/21821#discussion_r887403910